lang/python/python-more-itertools/Makefile

```diff
@@ -0,0 +1,43 @@
+#
+# Copyright (C) 2019 CZ.NIC, z. s. p. o. (https://www.nic.cz/)
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=python-more-itertools
+PKG_VERSION:=7.2.0
+PKG_RELEASE:=1
+
+PYPI_NAME:=more-itertools
+PKG_HASH:=409cd48d4db7052af495b09dec721011634af3753ae1ef92d2b32f73a745f832
+PKG_BUILD_DIR:=$(BUILD_DIR)/more-itertools-$(PKG_VERSION)
```

Comment:
Setting `PKG_BUILD_DIR` shouldn't be necessary.

Reply:
Removed